integration_test/tests/wallet.rs

```diff
@@ -558,6 +558,15 @@ fn wallet__send_to_address__modelled() {
     model.unwrap();
 }
 
+#[test]
+fn wallet__set_tx_fee() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    let fee_rate = FeeRate::from_sat_per_vb(2).expect("fee_rate");
```

Comment:
Since you have to rebase anyways can you change this to 
```suggestion
    let fee_rate = FeeRate::from_sat_per_vb(2).expect("2 sat/vb is valid");
```

The idea being the expect message is basically a comment to those reading the code and is never going to be hit at runtime.